api/institutions/views.py

```diff
@@ -8,18 +9,24 @@
 from framework.auth.oauth_scopes import CoreScopes
 
 from website.models import Node, User, Institution
+from website.util import permissions as osf_permissions
 
 from api.base import permissions as base_permissions
 from api.base.filters import ODMFilterMixin
 from api.base.views import JSONAPIBaseView
 from api.base.serializers import JSONAPISerializer
 from api.base.utils import get_object_or_error
+from api.base.parsers import (
+    JSONAPIRelationshipParser,
+    JSONAPIRelationshipParserForRegularJSON,
+)
+from api.base.exceptions import RelationshipPostMakesNoChanges
 from api.nodes.serializers import NodeSerializer
 from api.users.serializers import UserSerializer
 
 from .authentication import InstitutionAuthentication
-from .serializers import InstitutionSerializer
-
+from .serializers import InstitutionSerializer, InstitutionNodesRelationshipSerializer
+from .permissions import UserIsAffiliated
```

Comment:
I know we don't always do it, but generally prefer absolute over relative module paths